python_modules/event-pipeline-demo/event_pipeline_demo/configs_spark.py

```diff
@@ -0,0 +1,1686 @@
+                                            is_optional=True,
+                                        ),
+                                        'filters': Field(
+                                            String,
+                                            description='''Spark UI: Comma separated list of filter class names to apply to the Spark Web UI. The filter should be a standard javax servlet Filter. Filter parameters can also be specified in the configuration, by setting config entries of the form spark.<class name of filter>.param.<param name>=<value> For example: spark.ui.filters=com.test.filter1 spark.com.test.filter1.param.name1=foo spark.com.test.filter1.param.name2=bar''',
+                                            is_optional=True,
+                                        ),
+                                    }
+                                )
+                            ),
+                            'worker': Field(
+                                PermissiveDict(
+                                    fields={
+                                        'ui': Field(
+                                            PermissiveDict(
+                                                fields={
+                                                    'retainedExecutors': Field(
+                                                        String,
+                                                        description='''Spark UI: How many finished executors the Spark UI and status APIs remember before garbage collecting.''',
+                                                        is_optional=True,
+                                                    ),
+                                                    'retainedDrivers': Field(
+                                                        String,
+                                                        description='''Spark UI: How many finished drivers the Spark UI and status APIs remember before garbage collecting.''',
+                                                        is_optional=True,
+                                                    ),
+                                                }
+                                            )
+                                        )
+                                    }
+                                )
+                            ),
+                            'sql': Field(
+                                PermissiveDict(
+                                    fields={
+                                        'ui': Field(
+                                            PermissiveDict(
+                                                fields={
+                                                    'retainedExecutions': Field(
+                                                        String,
+                                                        description='''Spark UI: How many finished executions the Spark UI and status APIs remember bef
```

Comment:
if we want these to be wrapped in a more friendly way, there is this https://github.com/dagster-io/dagster/blob/master/python_modules/dagster-airflow/dagster_airflow/utils.py#L30 which i have been meaning to pull into dagster for some time

Reply:
yeah that'd be a good step. probably needs to be wrapped to > 120 since this is so crazy nested, but I'll plan to do a follow-up PR on this one